xclim/indices/_threshold.py

```diff
@@ -1050,21 +983,20 @@ def growing_season_end(
         Mean daily temperature.
     thresh : Quantified
         Threshold temperature on which to base evaluation.
-    mid_date : str
+    mid_date : str, optional
         Date of the year after which to look for the end of the season. Should have the format '%m-%d'.
     window : int
         Minimum number of days with temperature below threshold needed for evaluation.
     freq : str
         Resampling frequency.
-    op : {"<", "<=", "lt", "le"}
-        Comparison operation. Default: "<".
+    op : {">", ">=", "gt", "ge"}
+        Comparison operation. Default: ">". Note that this comparison is what defines the season.
+        The end of the season happens when the condition is NOT met for `window` consecutive days.
```

Comment:
This makes much more sense. The fact that it was inverted for one release shouldn't be so bad. Behaviour is unchanged.